src/components/joinevent.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import {
+    // eslint-disable-next-line no-unused-vars
+    withRouter, BrowserRouter as Router, Route, NavLink, Switch,
+} from 'react-router-dom';
+import { fetchEvents, fetchUserEvents } from '../actions';
+import Error from './error';
+
+// Displays list of events user is a part of
+class JoinEvent extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            eventList: [],
+            userEventList: [],
+        };
+    }
+
+    componentDidMount() {
+        // Get list of all events
+        this.props.fetchEvents((eventList) => { this.setState({ eventList }); });
+        // Get list of events for current user
+        this.props.fetchUserEvents(this.props.user.id, (userEventList) => {
+            const IDList = [];
+            // eslint-disable-next-line no-unused-vars
```

Comment:
No

Reply:
userEventList.entries() returns index and value together. switching [index, value] to value doesn't work, and doing const value of userEventList.entries() and referencing the second value of the array doesn't work either. i dont have a solution lol